src/webui/api/torrentscontroller.cpp

```diff
@@ -544,7 +544,7 @@ void TorrentsController::addAction()
     const TriStateBool addPaused = parseTriStateBool(params()["paused"]);
     const TriStateBool rootFolder = parseTriStateBool(params()["root_folder"]);
     const QString savepath = params()["savepath"].trimmed();
-    const QString category = params()["category"].trimmed();
+    const QString category = params()["category"];
```

Comment:
Is this set of changes related to the mentioned issue in commit message? If not, shouldn't you make it another commit?

Reply:
You're right, done.